README.md

```diff
@@ -1,3 +1,43 @@
 # Slack Data Collector
 
-Uses the Slack API to collect anonymized information about the community
\ No newline at end of file
+Uses the Slack API to collect anonymized information about the community
+
+## Installation instructions
+
+### OSX
+
+#### Step 1
+
+Make sure you have `Python` installed
+
+`brew install python3`
+
+#### Step 2
+
+Install needed packages
+
+`pip3 install nose`
+
+`pip3 install tox`
+
```

Comment:
I'm testing the setup on a fresh vm, those might not be needed if the user doesn't want to run the unit tests.